docs/dsl2.md

```diff
@@ -183,6 +183,26 @@ workflow {
 Optional params for a process input/output are always prefixed with a comma, except for `stdout`. Because `stdout` does not have an associated name or value like other types, the first param should not be prefixed.
 :::
 
+### Process config
```

Comment:
Ideally would like to merge #3793 first, then this would become part of the "Workflows" page which is more prominent.